tools/osbuilder/node-builder/azure-linux/package_install.sh

```diff
@@ -14,7 +14,6 @@ CONF_PODS=${CONF_PODS:-no}
 PREFIX=${PREFIX:-}
```

Comment:
We should generally deploy a secondary virtio-blk config file which can be used with containerd + tardev-snapshotter

Reply:
Good point. Added new config for virtio-blk based kata pod-sandboxing that uses tardev-snapshotter.